### PR TITLE
Add report to the file by merging earlier code. Closes #8

### LINKED ISSUE #8: [Student C] Save statistics to the file
Check instructions from the doc

### PR DESCRIPTION
**Pull request by Vajsent-123**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -152,7 +152,64 @@ def __setitem__(self, k, v):
...
+            super().__setitem__(k.lower(), v)
+
+
+    chars = CICounter(int)
+    for char in text_file:
+        chars[char] += 1
+    punctuationList = list(string.punctuation)
+
+
+    from collections import defaultdict
```
Comment (bpankowski) [inline main.py]: You don't have to call that multiple times

File: main.py [hunk 2]
```diff
@@ -152,7 +152,64 @@ def __setitem__(self, k, v):
...
+    class CICounter(defaultdict):
```
Comment (bpankowski) [inline main.py]: You don't have to call that multiple times

File: main.py [hunk 3]
```diff
@@ -152,7 +152,64 @@ def __setitem__(self, k, v):
...
+        def __getitem__(self, k):
+            return super().__getitem__(k.lower())
+
+        def __setitem__(self, k, v):
+            super().__setitem__(k.lower(), v)
+
+    global chars1
```
Comment (bpankowski) [inline main.py]: You don't have to call that multiple times as chars are already defined.

File: main.py [hunk 4]
```diff
@@ -35,8 +35,8 @@ def menu():
     elif choice=="8":
         if os.path.exists("text.txt") == True:
            os.remove("text.txt")
-        if os.path.exists("stats.txt") == True:
-           os.remove("stats.txt")
+        if os.path.exists("statystyka.txt") == True:
```
Comment (bpankowski) [inline main.py]: I know that doc is saying to save it as 'statystyki' file but keep a convention in naming and leave it as English word.

File: main.py [hunk 5]
```diff
@@ -152,7 +152,64 @@ def __setitem__(self, k, v):
...
+        chars1[char] += 1
+
+
+    stats.write("%d\n" % only_letters )
+    stats.write("%d\n" % only_words )
+    for i in punctuationList:
+        stats.write(i + " :" + str(chars[i])+"\n")
+    stats.write('No. sentences:' + str(chars1['.']))
+    stats.close()
+
```
Comment (bpankowski) [inline main.py]: leave just one free space

File: main.py (hunk 1 above)
Comment (Vajsent-123) [reply main.py]: fixed I moved it to start

File: main.py (hunk 2 above)
Comment (Vajsent-123) [reply main.py]: fixed I moved it to start

File: main.py (hunk 4 above)
Comment (Vajsent-123) [reply main.py]: changed to stats again

File: main.py [hunk 6]
```diff
@@ -152,6 +151,45 @@ def __setitem__(self, k, v):
...
+        def __setitem__(self, k, v):
+            super().__setitem__(k.lower(), v)
+
+    chars = CICounter(int)
+    for char in text_file:
+        chars[char] += 1
+    punctuationList = list(string.punctuation)
+
+    global chars1
```
Comment (bpankowski) [inline main.py]: I still think that we don't need it,

Comment (Vajsent-123) [reply main.py]: Ofcourse you were right. Fixed

File: main.py [hunk 7]
```diff
@@ -152,6 +151,45 @@ def __setitem__(self, k, v):
...
+
+    chars = CICounter(int)
+    for char in text_file:
+        chars[char] += 1
+    punctuationList = list(string.punctuation)
+    for i in punctuationList:
+        stats.write(i + " :" + str(chars[i])+"\n")
+
+
+    chars = CICounter(int)
```
[the remark this answers is not in the document]
Comment (Vajsent-123) [reply main.py]: fixed

**Pull request merged**